lime-runtime/src/main/java/com/here/gluecodium/common/LimeModelSkipPredicates.kt

```diff
@@ -64,6 +65,18 @@ object LimeModelSkipPredicates {
         return isSkipped ?: false
     }
 
+    private fun isSkippedByTagsOnPlatform(
+        limeElement: LimeNamedElement,
+        activeTags: Set<String>,
+        platformAttribute: LimeAttributeType
+    ): Boolean {
+        val isEnabled = hasTagsMatch(limeElement, platformAttribute, ENABLE_IF, activeTags)
+        if (isEnabled == false) return true
+
+        val isSkipped = hasTagsMatch(limeElement, platformAttribute, SKIP, activeTags)
+        return isSkipped ?: false
```

Comment:
Just `return isSkipped` ?

Reply:
`hasTagsMatch()` returns a `Boolean?`, not `Boolean`. So neither of your suggestions will work.